core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs

```diff
@@ -119,3 +206,91 @@ impl<T: TokenPriceAPI + Send + Sync> FeeTickerAPI for TickerApi<T> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use bigdecimal::ToPrimitive;
+    use std::env;
+    use zksync_types::{Address, Token, TokenId, TokenKind, TokenPrice};
+
+    #[tokio::test]
+    async fn should_return_one_for_rdoc() {
+        const DATABASE_URL: &str = "postgres://postgres@localhost/plasma";
+        const RDOC_SYMBOL: &str = "RDOC";
+        const RDOC_VALUE: u32 = 1;
+
+        env::set_var("DATABASE_URL", DATABASE_URL);
```

Comment:
are we sure that we need this? have we tried to run the tests with `zk` as they are supposed to be run?